index.js

```diff
@@ -163,17 +173,24 @@ function fastifyStatic (fastify, opts, next) {
       }
     } else {
       const globPattern = typeof opts.wildcard === 'string' ? opts.wildcard : '**/*'
+
+      // TODO: This won't work with an array of roots.
+      if (Array.isArray(sendOptions.root)) {
+        return next(new Error('wildcard option not currently supported with multiple root paths'))
+      }
+
       glob(path.join(sendOptions.root, globPattern), { nodir: true }, function (err, files) {
         if (err) {
           return next(err)
         }
         const indexDirs = new Set()
         const indexes = typeof opts.index === 'undefined' ? ['index.html'] : [].concat(opts.index || [])
         for (let file of files) {
+          // TODO: This won't work with an array of roots.
           file = file.replace(sendOptions.root.replace(/\\/g, '/'), '').replace(/^\//, '')
-          const route = encodeURI(prefix + file).replace(/\/\//g, '/')
+          const route = (prefix + file).replace(/\/\//g, '/')
```

Comment:
is this change that removed encodeURI tested somehow? I'm not sure if this was necessary in the first place, but if we don't know for sure we should assume that it was necessary.

Reply:
Well spotted. Not sure why that changed but it should indeed be `encodeURI`.